win_build_vs.bat

```diff
@@ -23,6 +23,7 @@ set builddir=%1
 set skiptests=OFF
 set cmake_build_type=Release
 set build_type=Release
+set build_platform=Win32
```

Comment:
When using with vcvarsall.bat, it sets an environment variable which makes msbuild default to x86 as the default platform, not Win32, so we use this to make sure we use the proper Win32 platform.